src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      data-qa="big"
+      method="post"
+      class="search__form"
+    >
+      <input
+        class="search__form__input big__input"
```

Comment:
```suggestion
        class="search__form__input"
```
pay attention on this point from [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)
![Screenshot from 2023-10-01 17-16-12](https://github.com/mate-academy/layout_search-bar-airbnb/assets/33029864/12ecacf9-1b34-4e06-a2f3-28484d95dde6)